src/components/VoiceAssistant.tsx

```
@@ -154,13 +154,31 @@ export const VoiceAssistant: React.FC<VoiceAssistantProps> = ({ onSearch, isSear
   };
 
-  const startListening = () => {
+  const startListening = async () => {
     if (!recognitionRef.current || isListening || isSearching) return;
 
     setError(null);
     setTranscript('');
     
     try {
+      // Check if getUserMedia is supported
+      if (navigator.mediaDevices && navigator.mediaDevices.getUserMedia) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)